File: Femis Assignments/Morning Snacks/ArmstrongRange.java
public class ArmstrongRange {
public static void main(String[] args) {
for (int number = 1; number <= 1000; number++) {
int actualNumber = number;
int digits = String.valueOf(number).length();
int sum = 0;

while (actualNumber > 0) {
int digit = actualNumber % 10;


int power = 1;
for (int i = 0; i < digits; i++) {
power *= digit;
}

sum += power;
actualNumber /= 10;
}

if (sum == number) {
System.out.println(number);
}
}
}
}

File: Femis Assignments/Morning Snacks/ArmstrongRange.js
for (let number = 1; number <= 1000; number++) {
let actualNumber = number;
let digits = number.toString().length;
let sum = 0;

while (actualNumber > 0) {
let digit = actualNumber % 10;
let power = 1;
for (let i = 0; i < digits; i++) {
power *= digit;
}

sum += power;
actualNumber = parseInt(actualNumber / 10);
}

if (sum === number) {
console.log(number);
}
}

